app/com/arpnetworking/play/configuration/ConfigurationHelper.java

```diff
@@ -111,6 +113,17 @@ public static FiniteDuration getFiniteDuration(final Config configuration, final
                 .getInstance(clazz);
     }
 
+    /**
+     * Resolves and converts a piece of a {@link Config} to a JSON string.
+     *
+     * @param config The config to be rendered as JSON.
+     * @param path The path to the sub-object to render as JSON.
+     * @return The JSON representation of that sub-object.
+     */
+    public static String toJson(final Config config, final String path) {
+        return config.resolve().getObject(path).render(ConfigRenderOptions.concise());
```

Comment:
I don't think you need to .resolve() anymore.  It should've already been done for you.